### ISSUE TITLE
Build Scoreboard

### ISSUE THREAD

**Issue by PixelOrange** [PixelOrange/MARS#28]
- [ ] All Time
- [ ] Yearly
- [ ] Monthly
- [ ] Daily

I was originally going to include weekly, but thinking about it I don't know if there's a good way to specify what a "week" is in python. I'm sure there is one that I don't know of, but I think 4 scoreboards is plenty.

**Comment by Snorrrlax** [comment]
If it's possible, having weekly ones would be quite satisfying though... Otherwise there's a jump for daily to monthly. I googled specifying a week in python and people seem to be talking about it, although I have no idea what it all means... That was helpful of me haha, not :P 

I take it it's not as simple as calculating it from 7 consecutive daily scoreboards? Or setting a timer that counts seven days?

**Comment by PixelOrange** [comment]
I'll look into it, but it's going to be the lowest of priorities of the scoreboards just because of the complexity.

**Comment by Snorrrlax** [comment]
Understandable :)

**Comment by Snorrrlax** [comment]
I've been thinking about how our [first version of the script](https://gist.github.com/changemyview/7cb2fec154d709cf584a) had a working monthly scoreboard in the sidebar, and it occurred to me that while the script has been rewritten and modularised, it is still the same language, and wondered if there is some pieces of code in there for the monthly scoreboard that can be used, or translated, here?

**Comment by PixelOrange** [comment]
Yes, I lean on deltabot 1.0 for old features that need to be reimplemented. I don't like how the old scoreboard pages worked, I have some ideas and I hope to have some time to work on it this week.

**Comment by Snorrrlax** [comment]
I guess we should decide whether to use the word scoreboard or leaderboard to keep it consistent. I've been using leaderboard so far but since you started using scoreboard I think I prefer it.

**Comment by Snorrrlax** [comment]
Also, are you planning on including this?: https://github.com/alexames/DeltaBot/issues/18

Because I've been talking to qtx and we're going to include it in the new CSS.

**Comment by PixelOrange** [comment]
Yes. I plan on doing the following:

* Set "points" CSS class to all flair types.

* Set highest delta earner with CSS class "highestpoints"

* Set monthly(?) winners as "top10points" and possibly something to distinguish the highest earner for that month as well.

**Comment by Snorrrlax** [comment]
Cool! :)
